project-base/storefront/components/Pages/Contact/ContactContent.tsx

```
@@ -50,74 +50,71 @@ export const ContactContent: FC = () => {
-                                </FormButtonWrapper>
-                            </FormBlockWrapper>
-                        </FormContentWrapper>
-                    </Form>
-                </FormProvider>
-            </Webline>
-        </div>
+        <Webline width="lg">
+            <h1 className="mb-4">{t('Write to us')}</h1>
+
+            {settingsData?.settings?.contactFormMainText !== undefined && (
+                <div className="mb-4" dangerouslySetInnerHTML={{ __html: settingsData.settings.contactFormMainText }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)